include/aie/AIE.td

```diff
@@ -1603,6 +1603,40 @@ def AIE_ObjectFifoCreateOp: AIE_Op<"objectFifo.createObjectFifo", []> {
   }];
 }
 
+def AIE_ObjectFifoRegisterExternalBuffersOp: AIE_Op<"objectFifo.registerExternalBuffers", []> {
+  let summary = "Registers external buffers to given object fifo to use in the shim tile";
+  let description = [{
+    The "aie.objectFifo.registerExternalBuffers" operation 
+
+    Example:
+    ```
+      %of_t70_t73 = AIE.objectFifo.createObjectFifo(%t70, %t73, 2) : !AIE.objectFifo<memref<64xi16>>
+      %buffer_in_0  = AIE.external_buffer 0x020100004000 : memref<512 x i16>
+      %buffer_in_1  = AIE.external_buffer 0x020100006000 : memref<512 x i16>
```

Comment:
Did you mean to include the physical address here? #183

Reply:
No, that's a leftover from before it was removed. Thank you for pointing it out!